HISTORY.rst

```diff
@@ -1,14 +1,18 @@
 Release History
 ===============
 
-..
-  Unreleased Changes
-  -----------------
+
+Unreleased Changes
+------------------
+* Updated ``MANIFEST.in`` to only include files that should be there in the
+  pygeoprocessing source distribution.  This fixes an issue where files
+  matching a variety of extensions anywhere in the pygeoprocessing directory
+  might be included with the source distribution.
```

Comment:
Ah, this answers and confirms my comment. Also, multiline comments!

Reply:
I know, right??  Super helpful.